### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by ikemsley**

### CODE REVIEW

**Comment by tomdos** [comment]
@ikemsley Can you check travis please?

File: al_aws.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+ *
+ * Helper class for lambda function utility and helper methods.
+ *
+ * @end
+ * -----------------------------------------------------------------------------
+ */
+
+const AWS = require('aws-sdk');
+const moment = require('moment');
+const async = require('async');
```
Comment (tomdos) [inline al_aws.js]: It looks we don't use it.

File: package.json [hunk 2]
```diff
@@ -24,7 +24,8 @@
     "async": "*",
     "moment": "^2.19.2",
     "request": "*",
-    "request-promise-native": "*"
+    "request-promise-native": "*",
```
Comment (tomdos) [inline package.json]: Are we using `request*` ?

File: package.json [hunk 3]
```diff
@@ -24,7 +24,8 @@
...
+    "al-collector-js": "git://github.com/alertlogic/al-collector-js#master"
```
Comment (tomdos) [inline package.json]: We don't use `al-collector-js` dependency anywhere. Can you remove it pls?

Comment (kkuzmin) [reply package.json]: +1, let's add it back when needed.

**Pull request merged**